src/graphql/queries/events.ts

```diff
@@ -1,24 +1,63 @@
 import gql from 'graphql-tag';
 
 export interface EventType {
+  id: string;
   name: string;
   eventDate: Date;
+  // ambulances: Ambulance[];
+  // hospitals: Hospital[];
```

Comment:
was this intentional?

Reply:
This was intentional because `Ambulance` and `Hospital` were undefined, but I've resolved this now.